pkgs/by-name/ju/julec/package.nix

```
@@ -93,6 +94,12 @@ clangStdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  passthru = {
+    # see doc/hooks/julec.section.md
+    hook = callPackage ./hook.nix { };
+    tests.setupHooks = callPackage ./test { };
```

We should flow the final package through, so the test will activate appropriately on overrides of this package also.

```suggestion
    hook = callPackage ./hook.nix { julec = finalAttrs.finalPackage; };
    tests.setupHooks = callPackage ./test { julec = finalAttrs.finalPackage; };
```

And maybe rename the test from `setupHooks` to `example` or `hello-jule` as that's what's being built.